components/app.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-detils';
+import SearchForm from './search-form';
+
+import '../src/styles/app.scss';
+
+const URL = 'https://mcr-codes-weather.herokuapp.com/forecast?city=';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: 'manchester',
+        country: '',
+      },
+    };
+  }
+
+  goFetchNow = city => {
+    fetch(`${URL}${city}`)
+      .then(data => data.json())
+      .then(data => {
```

Comment:
It looks like you didn't get round to adding the `.catch()` block on this request. When you do, see if you can incorporate the information you get from error responses into the message you are displaying to the user.